helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -239,7 +239,7 @@ internal fun OrtResult.processAllCopyrightStatements(
         copyrightGarbage = CopyrightGarbage(copyrightGarbage.toSortedSet())
     )
 
-    getProjectAndPackageIds().forEach { id ->
+    collectProjectsAndPackages().forEach { id ->
```

Comment:
commit-msg: Please indicate whether the change is intended to be non-functional.

Reply:
Done.